examples/md-flexible/src/Simulation.cpp

```diff
@@ -578,18 +642,39 @@ void Simulation::logMeasurements() {
     std::cout << "MFUPs/sec                          : " << mfups << std::endl;
 
     if (_configuration.dontMeasureFlops.value) {
-      LJFunctorTypeAbstract ljFunctor(_configuration.cutoff.value, *_configuration.getParticlePropertiesLibrary());
-      autopas::FlopCounterFunctor<ParticleType, LJFunctorTypeAbstract> flopCounterFunctor(
-          ljFunctor, _autoPasContainer->getCutoff());
-      _autoPasContainer->iteratePairwise(&flopCounterFunctor);
-
-      const auto flops = flopCounterFunctor.getFlops();
-
-      std::cout << "Statistics for Force Calculation at end of simulation:" << std::endl;
-      std::cout << "  GFLOPs                             : " << static_cast<double>(flops) * 1e-9 << std::endl;
-      std::cout << "  GFLOPs/sec                         : "
-                << static_cast<double>(flops) * 1e-9 / (static_cast<double>(simulate) * 1e-9) << std::endl;
-      std::cout << "  Hit rate                           : " << flopCounterFunctor.getHitRate() << std::endl;
+      if (_configuration.getInteractionTypes().count(autopas::InteractionTypeOption::pairwise)) {
+        LJFunctorTypeAbstract ljFunctor(_configuration.cutoff.value, *_configuration.getParticlePropertiesLibrary());
+        autopas::FlopCounterFunctor<ParticleType, LJFunctorTypeAbstract> flopCounterFunctor(
+            ljFunctor, _autoPasContainer->getCutoff());
+        _autoPasContainer->computeInteractions(&flopCounterFunctor);
+
+        const auto flops = flopCounterFunctor.getFlops();
+
+        std::cout << "Statistics for the Pairwise Force Calculation at end of simulation:" << std::endl;
+        std::cout << "  GFLOPs                             : " << static_cast<double>(flops) * 1e-9 << std::endl;
+        std::cout << "  GFLOPs/sec                         : "
+                  << static_cast<double>(flops * _iteration) * 1e-9 / (static_cast<double>(forceUpdatePairwise) * 1e-9)
+                  << std::endl;
+        std::cout << "  Hit rate                           : " << flopCounterFunctor.getHitRate() << std::endl;
+      }
+
+#ifdef MD_FLEXIBLE_FUNCTOR_AT
+      if (_configuration.getInteractionTypes().count(autopas::InteractionTypeOption::triwise)) {
+        ATFunctorTypeAbstract atFunctor(_configuration.cutoff.value, *_configuration.getParticlePropertiesLibrary());
+        autopas::FlopCounterFunctor3B<ParticleType, ATFunctorTypeAbstract> flopCounterFunctor(
+            atFunctor, _autoPasContainer->getCutoff());
+        _autoPasContainer->computeInteractions(&flopCounterFunctor);
+
+        const auto flops = flopCounterFunctor.getFlops();
+
+        std::cout << "Statistics for the 3-Body Force Calculation at end of simulation:" << std::endl;
```

Comment:
I'd either call this one the tri-wise force calculation or the other one the 2-Body one. Just a consistency thing ;)